### PR TITLE
Updated TERA content for special characters

### PR DESCRIPTION
**Pull request by mkerns1**
## Summary

- Removed unneeded hint text and updated content related to special character changes.

## Related issue(s)

- [Ticket 88537](https://app.zenhub.com/workspaces/10-10-health-apps-5fff0cfd1462b6000e320fc7/issues/gh/department-of-veterans-affairs/va.gov-team/88537)

## Acceptance criteria

- [x] Updated content based on the Figma designs

### CODE REVIEW

File: src/applications/ezr/locales/en/content.json [hunk 1]
```diff
@@ -54,14 +54,14 @@
...
   "military-service-other-exposure-dates-start": "Exposure start date",
   "military-service-other-exposure-dates-end": "Exposure end date",
   "military-service-other-exposure-title-2": "Other toxic exposure",
-  "military-service-other-exposure-description": "You selected that you were exposed to other toxins or hazards.",
-  "military-service-other-exposure-description-2": "Enter any toxins or hazards you\u2019ve been exposed to",
+  "military-service-other-exposure-description": "You selected that you think you may have been exposed to other toxins or hazards.",
+  "military-service-other-exposure-description-2": "Enter any toxins or hazards you think you may have been exposed to",
   "military-service-other-exposure-description-2-hint": "If you\u2019re listing multiple toxins or hazards, list them without commas or any other special characters",
-  "military-service-other-exposure-error-message": "You entered a character we can\u2019t accept. Remove any special characters like commas or dashes.",
+  "military-service-other-exposure-error-message": "You entered a character that we can't accept. You can use only these characters: . , ! ?",
```
Comment (coope93) [inline src/applications/ezr/locales/en/content.json:64]: Does `can't` need to be `can\u2019t`?

File: src/applications/ezr/locales/en/content.json [hunk 2]
```diff
@@ -54,14 +54,14 @@
...
+  "military-service-other-exposure-dates-description": "Enter any date range when you think you may have been exposed to other toxins or hazards. You don\u2019t need to have exact dates.",
   "military-service-other-exposure-dates-start": "Exposure start date",
   "military-service-other-exposure-dates-end": "Exposure end date",
   "military-service-other-exposure-title-2": "Other toxic exposure",
-  "military-service-other-exposure-description": "You selected that you were exposed to other toxins or hazards.",
-  "military-service-other-exposure-description-2": "Enter any toxins or hazards you\u2019ve been exposed to",
+  "military-service-other-exposure-description": "You selected that you think you may have been exposed to other toxins or hazards.",
+  "military-service-other-exposure-description-2": "Enter any toxins or hazards you think you may have been exposed to",
   "military-service-other-exposure-description-2-hint": "If you\u2019re listing multiple toxins or hazards, list them without commas or any other special characters",
```
Comment (coope93) [inline src/applications/ezr/locales/en/content.json:63]: Should this be removed since I don't see anywhere we are using it now?

**Review by coope93 — CHANGES_REQUESTED** [review]

File: src/applications/ezr/locales/en/content.json (hunk 2 above)
Comment (mkerns1) [reply src/applications/ezr/locales/en/content.json:63]: We use these in the `otherToxicExposureDetails.js` file.

Comment (coope93) [reply src/applications/ezr/locales/en/content.json:63]: I thought that was just deleted in this PR?

Comment (mkerns1) [reply src/applications/ezr/locales/en/content.json:63]: Nope it's overwritten. Still in use.

Comment (mkerns1) [reply src/applications/ezr/locales/en/content.json:63]: Ohhhhh sorry for some reason it looked like you were talking about the `"military-service-other-exposure-description"` and `"military-service-other-exposure-description-2"`. You're right the hint text can be deleted. I will push that right up.

**Review by coope93 — APPROVED** [review]

**Pull request merged**